Cargo.toml

```diff
@@ -131,16 +127,21 @@ x11rb = { version = "0.11.1", features = [
     "cursor",
     "xinput",
 ], optional = true }
-wayland-client = { version = "0.29.5", optional = true, features = [
-    "use_system_lib",
-] }
+wayland-client = { version = "0.29.5", optional = true }
```

Comment:
Should this be grouped with the other wayland dependencies below?

How much do you need to use the lower-level wayland-client library after the switch to smithay-client-toolkit? I do understand that the toolkit is just designed to abstract some of what wayland-client offers.

Reply:
These are just entirely unused dependencies. I've removed them